source/docs/articles/sites.md

```diff
@@ -1,33 +1,73 @@
+
+###Delete Site
+If you no longer need your site, you can remove it here.
+**Note**: This is a permanent change and once you delete it your site cannot be restored.
+
+###About Site
+This tab shows general information about your site, such as the framework, upstream, date it was created, and your current plan.
+
+###PHP Version
+The site default PHP version is applied to every environment unless a PHP version is explicitly selected for each environment. Learn more about [Toggling Between PHP Versions](/docs/articles/sites/settings/toggling-between-php-versions/).
+
+## Support
+If you need assistance, our expert Customer Support team is here to help make building and managing your sites easier. You can contact Support by opening a ticket from your Dashboard. Learn more about our [scope of support](/docs/articles/scope-of-support/).
+
+##Public
+You can set a common password for accessing this environment to add an extra layer of security to prevent unwanted access to this environment. Learn more about [Locking Your Site](/docs/articles/sites/security/locking-your-site/).
+
 ## Connection Information
-## Delete Site
-## Deploy
-## Domains
-## Errors
-## Jobs Indicator
+Here you can find the [SSH clone URL](docs/articles/local/starting-with-git/), [Database Connection Information](/docs/articles/local/accessing-mysql-databases/), and [SFTP](/docs/articles/sites/code/developing-directly-with-sftp-mode/) connection information.
+
+## Code
+The core of the Pantheon Workflow is to move code up from Dev to Test to Live and content down from Live to Test to Dev.
+
+- **Code** includes plugins, modules, themes, CSS, JS—anything that's under Git version control.
+- **Content** includes files not under Git version control, like images and pdfs, and the database.  
+
+Learn how to use the [Pantheon workflow](/docs/articles/sites/code/using-the-pantheon-workflow/) to manage your site's code.
+
 ## Merge
-## MultiDev
-## PHP Version Toggle
-## Security
-## Settings
-## SSL
+Visit this tab when you have commits you need to merge into another environment.
+
 ## Status
-## Support
-## Team
-## Visit Site
+Pantheon provides static site analysis as a service for your site to make best practice recommendations on site configurations and to help detect common problems. This mechanism does not perform requests on your site, and in doing so avoids the observer effect. It's non-intrusive, so no installation or configuration is required. Finally, it's completely automated for consistent reports and results. Learn more about our Launch Check for
+[Drupal sites](/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis/) or [WordPress sites](docs/articles/wordpress/launch-check-wordpress-performance-and-configuration-analysis/).
+
 ## Workflow
+Use this tab to clone database and files from an environment. Learn more about the [Pantheon Workflow](https://pantheon.io/docs/articles/sites/code/using-the-pantheon-workflow/).
```

Comment:
That link doesn't talk about the content workflow tool. Clone, import, export, wipe are the functions here.